### DISCUSSION TITLE
Embedded LanceDB in a Tauri (Rust) desktop app

### DISCUSSION THREAD

**Discussion by tleyden** [lancedb/lancedb#1152] [Q&A]
I'm creating a Tauri / Rust desktop app that needs vector embedding search, would LanceDB be a good fit?  From the description it sounds like it is designed to run in-process:

> Embedded within an existing backend (like your Django, Flask, Node.js or FastAPI application)  

I was planning to use sqlite + sqlite-vss, but at the moment the sqlite-vss rust bindings [don’t seem well supported yet](https://github.com/asg017/sqlite-vss/issues/50), so I'm looking for alternatives.

**Comment by AyushExel** [comment]
Yes LanceDB would be a great choice as it is designed to run in-process.
